org.mwc.debrief.legacy/src/Debrief/Wrappers/SensorWrapper.java

```diff
@@ -878,8 +878,6 @@ public static int mergeSensors(final Editable targetE, final Layers theLayers, f
 		for (int i = 0; i < subjects.length; i++) {
 			final SensorWrapper sensor = (SensorWrapper) subjects[i];
 			if (sensor != target) {

```

Comment:
@ayeshare - the next line has been deleted. Wasn't it doing something useful?

Reply:
No it wasnt doing anything useful.  The methods append(target) was being called and it was not implemented for SensorWrapper.  We removed implements Layer on TacticalWrapper, as none of the methods from layer was implemented on TacticalWrapper.

Reply:
This version of `append` is still in the file.

![image](https://user-images.githubusercontent.com/1108513/99531288-69b36900-299a-11eb-8cca-48ea30041b28.png)

This call was essential to add the sensor cuts from the _other_ sensor into the target sensor.